### PR TITLE
Add link to Pachyderm PFS volume driver plugin in plugins.md

### PR DESCRIPTION
**Pull request by bufdev**
Signed-off-by: Peter Edge peter.edge@gmail.com

This is a plugin we're developing right now and will be core to our future functionality. Wanted to add it to the documentation as another example :)

### CODE REVIEW

File: docs/extend/plugins.md [hunk 1]
```diff
@@ -59,6 +59,10 @@ The following plugins exist:
   which is written in Go and provides advanced storage functionality for many
   platforms including EC2, OpenStack, XtremIO, and ScaleIO.
 
+* The [Pachyderm PFS plugin](https://github.com/pachyderm/pachyderm/tree/master/src/cmd/pfs-volume-driver)
+  is a volume plugin written in Go that provides functionality to mount Pachyderm File System (PFS)
+  repositories at specific commits as volumes within Docker containers.
```
Comment (thaJeztah) [inline docs/extend/plugins.md]: Would be nice if you could move it above the "Rex Ray" plugin, to keep it in alphabetical order

**Comment by thaJeztah** [comment]
LGTM, one nit, that would be nice to have changed, but no show-stopper

**Comment by peter-edge** [comment]
@thaJeztah updated that :) no problem

**Comment by thaJeztah** [comment]
Great! LGTM

ping @SvenDowideit

**Comment by SvenDowideit** [comment]
LGTM

**Pull request merged**